spec/lib/authorization_spec.rb

```diff
@@ -32,11 +32,7 @@
         [[{subjects: users[0].hyper_tag_name, actions: "Vm:view", objects: vms[0].hyper_tag_name(access_policy.project)}], users[0].id, "Vm:view", 1],
         [[{subjects: users[0].hyper_tag_name, actions: "Vm:view", objects: vms.map { _1.hyper_tag_name(access_policy.project) }}], users[0].id, "Vm:view", 2],
         [[{subjects: users[0].hyper_tag_name, actions: "Vm:delete", objects: vms[0].hyper_tag_name(access_policy.project)}], users[0].id, "Vm:view", 0],
-        [[{subjects: users[0].hyper_tag_name, actions: "Vm:Firewall:create", objects: vms[0].hyper_tag_name(access_policy.project)}], users[0].id, "Vm:Firewall:delete", 0],
```

Comment:
We need these tests to confirm our ability to authorize multi-level actions. Instead of removing them, could you substitute them with `Postgres:Firewall:view` or `Postgres:Firewall:edit`?